castepinput/inputs.py

```diff
@@ -55,7 +55,7 @@ def get_file_lines(self):
                 # If a list/tuple is passed join into a string
                 if isinstance(value, (tuple, list)):
                     value = " ".join(map(str, value))
-                if value:
+                if value or value == False:
```

Comment:
Can you change this to `if value is not None` instead?
The only reason that we do this check is that some keywords simply stand on their own without any value needed.

Reply:
The way this is currently, this will allow "" or None to default to the standalone.  If we make it is not None, then the "" will be output.  Based on your examples I figured you wanted it to work this way.

Reply:
Ahh I see, yes we definitely want to `""` to work. Perhaps it should be
```
if value is not None and value != "":
```
to be more explicit of the special cases.  Using `==` for boolean comparison might have some caveats.